src/javascript/class-scanner.ts

```diff
@@ -361,43 +361,38 @@ class PrototypeMemberFinder implements Visitor {
 
   enterExpressionStatement(node: babel.ExpressionStatement) {
     if (babel.isAssignmentExpression(node.expression)) {
-      this._createMemberFromAssignment(node);
+      this._createMemberFromAssignment(node.expression,
+        getJSDocAnnotationForNode(node));
     } else if (babel.isMemberExpression(node.expression)) {
-      this._createMemberFromMemberExpression(node);
+      this._createMemberFromMemberExpression(node.expression,
+        getJSDocAnnotationForNode(node));
     }
   }
 
-  private _createMemberFromAssignment(node: babel.ExpressionStatement) {
-    if (!babel.isAssignmentExpression(node.expression)) {
+  private _createMemberFromAssignment(
+      node: babel.AssignmentExpression, jsdocAnn?: jsdoc.Annotation) {
```

Comment:
Why is the second arg optional, we always pass it?

If it's because it could be undefined, prefer `jsdoc.Annotation|undefined`, as that way we'll be less likely to accidentally leave off that argument.